models/vista3d/docs/README.md

```diff
@@ -4,7 +4,7 @@ own datasets. To reproduce the paper results, please refer to https://github.com
 
 # Installation Guide
 ```
-pip install "monai[fire]"
+pip install "monai[fire]" requests huggingface_hub
```

Comment:
Hi @surajpaib , thanks for the contribution.
Since the bundle content is changed, can you also update the `metadata.json` with a new version and changelog?
https://github.com/Project-MONAI/model-zoo/blob/dev/models/vista3d/configs/metadata.json#L5